tests/cypress/integration/Home/homePageWithWoo.cy.js

```diff
@@ -186,8 +196,13 @@ describe( 'Commerce Home Page- When WooCommerce is installed', () => {
 	} );
 
 	it( 'Verify next step "Add a Product"', () => {
-		cy.contains( 'Add a product' ).as( 'addProduct' ).should( 'exist' );
-		cy.get( '@addProduct' ).click();
+		waitForNextSteps();
+		cy.contains( '.nfd-grid.nfd-gap-4 ul li a', 'Add a product', {
```

Comment:
We were going to stop checking texts I think?

Reply:
In un-avoidable cases where text is the only unique locator we have still kept text identifier. For example **Next Steps** list elements (<li>) which are all having same html properties except the text.

Reply:
In that case I think we should check for pt_BR as well for these texts in un-avoidable cases

Reply:
Currently we are not running tests against pt_BR. In future yes, we must add some unique identifiers like id, data-test-id etc to such elements and make them independent of text.